adalib/ada/datasets/toys.py

```diff
@@ -371,8 +371,11 @@ def __init__(
         cluster_hash = xp.param_to_hash(cluster_params)
         transform_hash = xp.param_to_hash(transform)
         self.data_dir = os.path.join(cluster_hash, transform_hash)
+        root_dir = os.path.join(self.root, self.raw_folder)
+        if not os.path.isdir(root_dir):
+            os.makedirs(root_dir)
```

Comment:
`os.makedirs(root_dir, exists_ok=True)` would be shorter and more consistent with what is done in other places.

Reply:
done